core/common/src/thrift/file_system_worker.thrift

```diff
@@ -0,0 +1,39 @@
+namespace java alluxio.thrift
+
+include "common.thrift"
+include "exception.thrift"
+
+struct UFSCancelFileTOptions {
```

Comment:
should this be renamed to `CancelUFSFileTOptions` too?
and the below two TOptions